test/mailers/previews/user_mailer_preview.rb

```diff
@@ -112,6 +112,42 @@ def change_email
     new_email = "new_email"
     UserMailer.change_email(user.id, old_email, new_email)
   end
+  
+  # Sends email when collection item changes status
+  [:anonymous_unrevealed_collection, :anonymous_collection, :unrevealed_collection].each do |status|
```

Comment:
I don't think this loop is doing exactly what you want. For example, should the first method be more like the following?
```rb
def anonymous_unrevealed_collection_notification
    user, collection, item = anonymous_or_unrevealed_data(:anonymous_unrevealed_collection)
    newly_anonymous = true
    newly_unrevealed = true
    UserMailer.anonymous_or_unrevealed_notification(
      user.id, item.id, collection.id,
      anonymous: newly_anonymous, unrevealed: newly_unrevealed
    )
end
```

Otherwise, I think the `status` needs to be in the method name to avoid new definitions overriding previous ones (I only see `anonymous_unrevealed_collection_notification` once on /rails/mailers for example)

Reply:
When adjusting this method, please also keep the guidelines for naming the mailer previews in mind: https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#user-content-Mailer_previews